framework-plugins/lisk-framework-dashboard-plugin/src/ui/components/CopyText/CopyText.module.scss

```diff
@@ -0,0 +1,11 @@
+.clickable {
+	cursor: pointer;
+}
+
+.clickableRow {
+	float: left;
```

Comment:
you can use `flex` here

Reply:
when using flex, Text and Icon components don't align side by side and icon goes to the next line

Reply:
the flex needs to be added to the container